app/controllers/comments_controller.rb

```diff
@@ -0,0 +1,45 @@
+class CommentsController < ApplicationController
+  before_action :authenticate_user!
+
+  def create
+    @talk = Talk.find(params[:talk_id])
+    @comment = Comment.new(talk_id: @talk.id, content: params[:comment][:content], user_id: current_user.id)
+    if @comment.save
+      flash[:notice] = 'Thanks for commenting'
+    else
+      @comment.errors.full_messages.each do |error|
```

Comment:
Unused block argument - `error`. You can omit the argument if you don't care about it.